guides/v2.3/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.md

```diff
@@ -1 +0,0 @@
-../../../v2.2/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.md
```

Comment:
Hi @abrarpathan19.
This change is also applicable for 2.2 (https://github.com/magento/magento2/blob/2.2/app/code/Magento/Customer/view/base/ui_component/customer_form.xml)
Please, make the changes directly in v2.2 of the documentation instead of introducing a new in v2.3 scope.

Thank you

Reply:
Also, please note that the same change is applicable for v2.1